Apps/VisionOS/VisionOS/ViewUtils/InlineNavigationLink.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+enum InlineNavigationLink: String, Codable, CaseIterable {
```

Comment:
What does "Inline" in the data type name mean?

Reply:
I updated the PR description. The name is coming from the fact that these links will appear as a markdown inline links. Something like [this](https://github.com/customerio/customerio-ios/pull/606) link which is just a clickable word(s) within a continuous line of text